src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -61,45 +61,60 @@ class ItemRecognition extends Component {
     );
   };
 
+  setSuggestions = (items, index) => {
+    const suggestions = [];
+    while (suggestions.length < 3) {
+      if (items[index].id === 'unknown') continue;
+      suggestions.push(this.props.storeList[items[index].id]);
+      index++;
+    }
+    this.props.setSuggestions(suggestions);
+  };
+
   handleImg = img => {
     if (this.success) return;
+
     this.model.predict(img).then(async items => {
       const item = items[0];
       if (
-        (item.value > ML_THRESHOLD &&
-          item.id !== 'unknown' &&
-          !this.props.prediction) ||
-        (Date.now() - this.scanningStartTime) / 1000 > TIMEOUT_IN_SECONDS
+        item.value > ML_THRESHOLD &&
+        item.id !== 'unknown' &&
+        !this.props.prediction
       ) {
+        // Item recognised
```

Comment:
Was meaning to mention this a while ago, could we assign this to a variable instead of a comment so it's the code that's doing the explaining?

```
const isItemRecognised = item.value > ML_THRESHOLD && item.id !== 'unknown' && !this.props.prediction

if (isItemRecognised) {
}
```